doc/internal/man3/DER_w_begin_sequence.pod

```diff
@@ -0,0 +1,48 @@
+=pod
+
+=head1 NAME
+
+DER_w_begin_sequence, DER_w_end_sequence
```

Comment:
Are you intending to add sequence/set(choice?) later?

Reply:
Yes